docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Store architecture use cases"
+slug: "store-architecture-use-cases"
+hidden: false
+createdAt: "2024-05-23T13:08:55.338Z"
+updatedAt: "2024-05-23T13:08:55.338Z"
+excerpt: "Learn how our store architecture models are tailored to meet diverse business needs."
+seeAlso:
+ - "/docs/guides/understanding-vtex-reference-architectures"
+ - "/docs/guides/store-architecture"
+---
+
+Store architecture communicates project vision, translating business needs into software requirements, and offering references for technical and business teams. This guide will explore the main VTEX reference architectures to learn how the VTEX platform integrates with different digital commerce strategies.
+
+>ℹ️ The diagrams were conceived according to the [Understanding VTEX reference architectures](https://developers.vtex.com/docs/guides/understanding-vtex-reference-architectures) guide.
+
+## Business-to-customer (B2C)
+
+The business-to-customer (B2C) is the most common business model in VTEX and the base for the other business models.
+
+The diagram below shows the main components of a B2C commerce implementation, highlighting in pink the ones offered out-of-the-box (OOTB) by VTEX, also showing the other parts that may come from customers’ internal systems or third-party providers.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2c.png)
+
+>ℹ️ This diagram represents one single VTEX account and assumes that the customer is using the VTEX storefront solution.
```

Comment:
Which VTEX storefront solution? Store Framework? Legacy Portal CMS? Fast Store? Any of those?